tests/tests_unit/test_utils/test_auxiliary.py

```diff
@@ -228,3 +229,27 @@ class TestFastDictLoad:
     def test_load(self, item, expected):
         get_accepted_params.cache_clear()  # For good measure
         assert expected == fast_dict_load(MyTestResource, item, cognite_client=None)
+
+
+class TestLoadDictOrStr:
+    @pytest.mark.parametrize(
+        "input, expected",
+        (
+            ({"foo": "bar"}, {"foo": "bar"}),
+            ('{"foo": "bar"}', {"foo": "bar"}),
+            ("foo: bar", {"foo": "bar"}),
+        ),
+    )
+    def test_load_dict_or_str(self, input, expected):
+        assert expected == load_dict_or_str(input)
+
+    @pytest.mark.parametrize(
+        "input",
+        (
+            "foo",
+            100,
+        ),
+    )
```

Comment:
The longest oneliner I've seen 😜

Reply:
I always expect it to just get auto-reformated by some formatting tool 🤣